package.json

```diff
@@ -29,6 +29,7 @@
   "author": "Stephen Sawchuk",
   "license": "MIT",
   "dependencies": {
+    "clang-format": "^1.1.0",
```

Comment:
Do we need this as a dep and not a devDep?

Reply:
Good catch, done